pandas/_libs/tslibs/period.pyx

```diff
@@ -1241,6 +1241,24 @@ cdef class _Period(object):
 
     @property
     def hour(self):
+        """
+        Get hours of a day that a Period falls on.
+
+        Returns
+        -------
+        int
+
+        See Also
+        --------
+        Period.minute : Get the minute of hour
+        Period.second : Get the second of hour
+
+        Examples
+        --------
+        >>> p = pd.Period("2018-03-11 13:03:12.050000")
+        >>> p.hour
+        13
+        """
```

Comment:
Can you add an example with a period longer than a day, to show that it uses 0?

```
In [4]: pd.Period("2017-01-01", freq="M").hour
Out[4]: 0
```

Reply:
can I add this too 
>>pd.Period("2017-01-01", freq="M").hour
0

Reply:
Yes, please do (with the `>>>` like you've done for the other).